tbx/project_styleguide/templates/patterns/pages/errors/404.html

```diff
@@ -1,11 +1,24 @@
 {% extends "patterns/base_page.html" %}
 
 {% block title %}Page not found{% endblock %}
+
+{% block body_class %}template-404-page mode-dark{% endblock %}
+
+{% block breadcrumbs %}{% endblock %}
+
 {% block content %}
-    <h1>
-        Err...
-    </h1>
-    <a href="/">
-        Take me home
-    </a>
+    <div class="grid mb-spacerLarge mt-spacerSmall streamfield text-center md:mt-spacerMedium xlg:mt-spacerHalf xxlg:mb-spacer lg:relative">
```

Comment:
Praise: good idea to use utility classes here rather than creating a specialised component.